seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/common/HttpClient.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
How about use the ability of `http-connector-base` instead of this?

Reply:
> Just like `seatunnel-formats`, we will have `seatuennl-client` as parent module and `seatunnel-client-http` as child module

`seatunnel-client` is easily misunderstood. I think use `http-connector-base` is better. However, I suggest that we can iterate it in the feature.